src/include/daos_srv/ras.h

```diff
@@ -41,6 +41,7 @@
 	X(RAS_ENGINE_DIED,		"engine_died")			\
 	X(RAS_ENGINE_ASSERTED,		"engine_asserted")		\
 	X(RAS_ENGINE_CLOCK_DRIFT,	"engine_clock_drift")		\
+	X(RAS_CORRUPTION_DETECTED,	"corruption_detected")		\
```

Comment:
probably want to rename to RAS_POOL_.. here to follow suite

Reply:
Will fix, thank you.